### ISSUE TITLE
pay 1.1.0 attempts to access "minimumButonWidth" and pay-ios 1.0.9 "minimumButtonWidth" on RawApplePayButton

### ISSUE THREAD

**Issue by np-eazy** [google-pay/flutter-plugin#237]
which I think is the root cause for the following error on my iOS app when attempting an Xcode build: "../../../.pub-cache/hosted/pub.dev/pay-1.1.0/lib/src/widgets/apple_pay_button.dart:35:38: Error: Member not found: 'minimumButonWidth'."

This typo has only started causing blocking errors when when updating flutter-stripe from 3.0 to 9.0.0^ (my app has not been maintained in months), and I think the build started actually catching it. 
- It's come up in the past before but mostly due to caching errors. 
- pay and pay-ios have been on 1.1.0 (taking to 1.1.2 causes other package problems) and 1.0.9 respectively for a while on my app, and I've been able to make updates elsewhere. 
- I only use pay to make Pay and PayProvider instances. I don't actually create any ApplePayButtons explicitly in my code but it seems that there are adjacent classes being used.

What I've tried so far:
- flutter pub cache repair: didn't really do anything
- Manually bump pay to 1.1.2: this opens a can of worms with linking and other dependencies needing upgrades/overrides so I'm avoiding this for now
- avoid using the Pay class altogether
- Changed the following code to use fromJsonString rather than fromAsset, which my IDE notified me was not recommended anymore:
```
Pay? googlePayClient;

Future<void> initializeGooglePayClient() async {
  String configsString = environment.googlePayPaymentProfile;
  googlePayClient = Pay({
    PayProvider.google_pay: PaymentConfiguration.fromJsonString(configsString)
  });
}
```

I'm surprised that I cannot find this error anywhere else on the Internet or StackOverflow. Maybe it's something with my cache. It has been an elusive error that popped up once when I was first learning to set up Flutter to work on this app, but that was fixed by running "flutter clean && flutter pub upgrade && pod update".

Right now I am thinking of forking pay 1.1.0 to manually resolve the typo, but I'm sure there's a better way to do this.

**Comment by emondd4** [comment]
Did This Issue Solved?

**Comment by np-eazy** [comment]
Sorry, forgot to mention that I've resolved this issue. Upgrading pay 1.1.2 was blocked by flutter-svg needing to be 2.0+, which came down to some linking issues that were fixed with changing the flags in the project.pbx file. 

That also mostly explains my original user-level error of the Stripe credit card options pop-up not rendering, which prompted the flutter-stripe upgrade/migration; I think the old flutter-svg dependency couldn't support the new payment sheet components, which were effectively invisible but still tappable.
